### ISSUE TITLE
REM mextension issue?

### ISSUE THREAD

**Issue by TharshanThana** [riscv/riscv-tests#113]
Can the remainder be negative?

**Comment by aswaterman** [comment]
Yes, see this test: https://github.com/riscv/riscv-tests/blob/master/isa/rv32um/rem.S

Like the C99 modulus operator, REM takes the sign of the dividend when the signs of the divisor and dividend differ.
